src/Service.Tests/Unittests/SerializationDeserializationTests.cs

```diff
@@ -305,6 +369,45 @@ private void InitializeObjects()
             _databaseStoredProcedure.StoredProcedureDefinition.Parameters.Add("Id", _parameterDefinition);
         }
 
+        /// <summary>
+        /// During serialization we add TypeName to the DatabaseObject based on its type.
+        /// If the code refactor then this change will catch that as GraphQL workload would be impacted.
```

Comment:
This sentence is unclear.
- If which code refactors? do you mean the JsonConverter you wrote for `System.Type`?
- and which change will catch the refactor? do you mean this test will catch a refactor?

Reply:
I have updated the documentation here now - added how typename currently looks. 
by code refactors I mean DAB codebase changes and changes the location of DatabaseObject.

![image](https://github.com/Azure/data-api-builder/assets/82844317/0cf801e4-a60b-44c5-a381-1a72194fc479)